backend/src/controllers/tests/getAchievments.ts

```diff
@@ -18,14 +18,16 @@ const getAchievments = async (
 
     const clientCourse = await getClientCourseProvider(clientCourseId);
     const { course } = clientCourse;
-    if ('technology' in course && clientCourse.status === CourseStatus.successful) {
-      const userSkills: { skills: ISkill[] } = await getUserSkills(userId);
-      const { oldSkills, newSkills } = specifySkills(userSkills.skills, course.technology);
+    const { technologies: techsToAchieve } = course;
+
+    if (clientCourse.status === CourseStatus.successful) {

```

Comment:
Rename please `CourseStatus` to the `COURSE_STATUSES` because it's not clear at first read that is a constant.

Reply:
The only one problem we're running into by doing that is lint enum naming error 😞. And it's not a good idea to disable this rule. I can disable it only for this file btw. Should I do that?

Reply:
Okay, let's leave as it is